views/admin.pug

```diff
@@ -4,10 +4,16 @@ block content
   h1= title
   p Welcome to #{title} - you are in admin mode
   a(href='#') Add new book
+  h4 Pages:
+  - for(var i = 0; i < `${count}`; i++) {
```

Comment:
you can do iteration a little easier in Pug, I don't know if something like this would be useful, or cleaner, code-wise ---> https://pugjs.org/language/iteration.html
``` 
  var  index = 0
  while index < `${count}` 
     index++